test/webview/SampleView.test.ts

```diff
@@ -233,6 +233,22 @@ describe('SampleView', () => {
             samplePageHtml.should.contain(`<button disabled class="open-button" onclick="openFile('applications','TypeScript Application')">Open Locally</button>`); // Disabled open button
         });
 
+        it('should display everything excluding the readme when there is no internet connection and the repository has not been cloned', async () => {
+            const outputAdapterSpy: sinon.SinonSpy = mySandBox.spy(VSCodeBlockchainOutputAdapter.instance(), 'log');
+            const repositoryRegistryExistsStub: sinon.SinonStub = mySandBox.stub(RepositoryRegistry.prototype, 'exists').returns(false);
```

Comment:
you don't need to stub this just make sure there is nothing in the registry

Reply:
Fixed